docs/roles/cloudera_manager/repo.md

```diff
@@ -0,0 +1,62 @@
+
+# Cloudera Manager Repository
+
+## Install the default Cloudera Manager version
+
+```
+(no extra vars)
+```
+
+With no variables overridden, the default behaviour is to install the latest or recommended version of Cloudera Manager (currently `7.0.3`). The version number is specified as `cloudera_manager_version` in the role default variables (`roles/cloudera_manager/repo/defaults/main.yml`).
+
+## Install a custom Cloudera Manager version (public archive, not paywalled)
+
+```
+cloudera_manager_version: 6.3.1
+```
+
+The version specified in `cloudera_manager_version` is used to construct repository URLs from the public `archive.cloudera.com` repository.
+
+## Install a custom Cloudera Manager version (public archive, paywalled)
+
+```
+cloudera_manager_version: 6.3.3
```

Comment:
The preferred method now is to supply the path to the `license_file` in the profile or definition; these methods also work but are more complicated

Reply:
Fixed